src/global-setup.js

```diff
@@ -1,18 +1,10 @@
 /* eslint-disable import/extensions */
 
 import mock from "jest-mock";
-import { expect } from "expect";
+import { jestExpect as expect } from "@jest/expect";
 import snapshot from "jest-snapshot";
```

Comment:
unused

Reply:
Thanks! Fixed in 9b0a402ca1832ac14c3aa4453969fbeabb6dd187